.github/workflows/test-windows.yml

```diff
@@ -31,5 +31,9 @@ jobs:
           cache: 'yarn'
       - name: Install Dependencies
         run: yarn --frozen-lockfile
-      - name: Run Windows tests
+      - name: Build prerequisites
+        run: yarn workspace modular-types build
+      - name: Run Windows tests (part 1)
         run: yarn test esmView.test.ts
+      - name: Run Windows tests (part 2)
+        run: yarn test workspace-resolver
```

Comment:
Can this be combined in the same step? I think you can simply do

`yarn test esmView.test.ts workspace-resolver`